packages/ethereum-contracts/.solcover.js

```diff
@@ -5,6 +5,11 @@ module.exports = {
     skipFiles: [
         "mocks/",
         "apps/SuperAppBase.sol",
```

Comment:
what's the role of SuperAppBase nowadays

Reply:
I believe it is used by the `SuperAppBaseCFA` and it's usually overridden, we could create two simple SuperApp's  which inherits from the respective bases and override the functions, but just use super.function, although not sure if what we are testing here is interesting.